views/index.md

```diff
@@ -14,6 +14,8 @@ now under the lead of Iain Buclaw, who has been steering the project since
 2009 with the assistance of its contributors.  Without them, the project
 would not have been nearly as successful as it has been.
 
+GDC was merged into the upstream GCC tree in GCC 9.0 and continues to be developed and distributed as part of GCC.
```

Comment:
You can use `gqq` in vim to break up the line, but I'm not really too bothered about line lengths.